static/src/stylesheets/inline/article-photo-essay-garnett.scss

```diff
@@ -1,268 +1,175 @@
+                }
             }
         }
+    }
 
-        &.element--supporting {
-            .caption {
-                max-width: 100%;
+    .photo-essay-block-quote {
+        font-style: normal;
+        display: inline;
+        float: left;
+
+        .inline-garnett-quote.inline-icon {
+            position: relative;
+            top: $gs-gutter/2;
+            svg {
+                width: 50px;
+                height: auto;
             }
         }
 
-        &.element--halfWidth {
-            width: 49%;
-            float: left;
-            margin-top: 0;
-            margin-bottom: 2%;
-
-            .inline-expand-image {
-                display: none;
-            }
-
-            &:before {
-                content: '';
-                display: block;
-                width: 2px;
-                top: 0;
-                bottom: 0;
-                z-index: 300;
-                position: absolute;
-                background-color: #ffffff;
-            }
-
-            // Makes sure that elements which come after the block of images are no longer floating
-            & + p, & + blockquote, & + .element-pullquote:not(.element--halfWidth) {
-                clear: both;
-                padding-top: $gs-baseline / 2;
-            }
-            // changes the padding for a half width quote after directly after a half width image
-            & + .element-pullquote.element--halfWidth {
-                margin-top: 0;
-            }
+        .quoted__contents {
+            @include fs-headlineGarnett(2);
+            margin: 0;
+            font-weight: 900;
 
-            & + .element-image:not(.element--halfWidth) {
-                clear: left;
+            @include mq($from: desktop) {
+                @include fs-headlineGarnett(4);
+                font-weight: 900;
             }
         }
-        &.element--halfWidth.half-width-odd {
-            margin-right: 2%;
-        }
     }
 
-    .element-pullquote--photo-essay.element--supporting {
-        .pullquote-paragraph {
-            text-indent: 0;
-        }
+    .element--halfWidth.element-pullquote--photo-essay {
+        width: 45%;
+        margin-right: $gs-gutter;
+        border-bottom: 0;
+        margin-top: 0;
+        clear: none;
+        margin-left: 0;
+        margin-bottom: $gs-baseline/2;
+        border-top: 1px solid $brightness-86;
+        vertical-align: top;
     }
 
-    .element-pullquote--photo-essay {
-        .pullquote-paragraph {
-            color: $sport-dark;
-            font-size: 26px;
-        }
+    .element-pullquote svg {
+        width: 18px;
+    }
 
-        //important because it is important elsewhere
-        .pullquote-cite {
-            color: $brightness-46 !important;
-            font-size: 26px;
-        }
+    .element-image.element--showcase {
+        margin-bottom: $gs-baseline;
+    }
+
+    .from-content-api ul:not(.submeta__links):not(.social--bottom):not(.syndication--bottom) {
```

Comment:
This scares me, but is familiar. It would be ideal if we could specifically target the exact uls